src/reference/common_errors.md

```diff
@@ -54,35 +76,47 @@ Caido allows 5 minutes of deviation between the "real" time and your computer ti
 
 ### Resolution for Windows
 
+<details>
+
 ::: info
 Time synchronization instructions for Windows can be found [here](https://www.majorgeeks.com/content/page/synchronize_clock_with_an_internet_time_server.html).
 :::
 
 1. Right-click on the clock.
-1. `Adjust date/time`.
-1. Go to `Date & Time` in `Setting`.
-1. Click `Sync now`.
+2. Select `Adjust date and time`.
+3. Go to `Date & Time` in `Settings`.
+4. Click `Sync now`.
```

Comment:
keep them all as 1, they are rendered correctly at runtime and it avoids changes if we need to add steps